concourse/setup.py

```diff
@@ -560,6 +564,38 @@ def deploy_delaying_proxy(
     ingress_helper.replace_or_create_ingress(namespace, ingress)
 
 
+def deploy_webhook_dispatcher(
+    concourse_cfg: ConcourseConfig,
+    webhook_dispatcher_cfg: WebhookDispatcher,
+    namespace: str,
+):
+    not_none(concourse_cfg)
+    not_empty(namespace)
+
+    ctx = kube_ctx
+
+    service_helper = ctx.service_helper()
+    deployment_helper = ctx.deployment_helper()
+    namespace_helper = ctx.namespace_helper()
+    ingress_helper = ctx.ingress_helper()
+
+    namespace_helper.create_if_absent(namespace)
+
+    webhook_service = generate_webhook_dispatcher_service()
+    webhook_deployment = generate_webhook_dispatcher_deployment(
```

Comment:
general remark: would it make sense to separate the creation of deployment manifests and their appliance (to better support local development scenarios)? for a first version, it is certainly ok to not do this.